pegasus/sites.v3/code.org/public/css/common.css

```diff
@@ -411,9 +411,9 @@
   padding-right: .5em;
 }
 
-button {
+/*button {
```

Comment:
Code should not be commented out without an explanation for why it's commented out.  Usually it should just be deleted.

In this case, though, it's probably more complicated.  The margin here was added so that adjacent buttons have vertical space when stacked vertically on narrow layouts, as part of our responsive implementation.  We've usually had to fix cases where the extra space wasn't desired.  @Erin007 can give more information.

Reply:
Yes, for example https://github.com/code-dot-org/code-dot-org/pull/14157 readjusts the alignment of the buttons that shifted when we didn't want them to after we made those pages responsive. To my understanding, there are more cases where we do want this extra space than where we don't, so we need to keep this top margin styling.

Reply:
Perhaps we can have a new CSS class for buttons which don't get this extra spacing and apply it to the exception buttons such as these?